Fix: Loading fixtures, does not update sequence (Postgres) (#20346)
yiisoft/yii2: framework/UPGRADE.md

version B between A and C, you need to follow the instructions
for both A and B.

Upgrade from Yii 2.0.52
-----------------------

> * There was a bug when loading fixtures, it would not reset the table sequences. If you used a work-around or if you depended on this beehaviur, you are advised to review your code.

Reviewer: ```suggestion
* There was a bug when loading fixtures into PostgreSQL database, the table sequences were not reset. If you used a work-around or if you depended on this behavior, you are advised to review your code.
```
